sentinel/src/main/java/gov/va/api/health/sentinel/LabBot.java

```
@@ -219,13 +220,32 @@ private void results(ExecutorService ex, List<Future<?>> futures) throws Interru
               () -> {
                 UserCredentials userCredentials =
                     UserCredentials.builder().id(userId).password(config.userPassword()).build();
-                VaOauthRobot bot =
-                    makeLabBot(userCredentials, config.baseUrl(), config().credentialsMode());
-                labBotUserResultList.add(
-                    LabBotUserResult.builder()
-                        .user(userCredentials)
-                        .tokenExchange(bot.token())
-                        .build());
+                try {
+                  VaOauthRobot bot =
+                      makeLabBot(userCredentials, config.baseUrl(), config().credentialsMode());
+                  labBotUserResultList.add(
+                      LabBotUserResult.builder()
+                          .user(userCredentials)
+                          .tokenExchange(bot.token())
+                          .build());
+                } catch (Exception e) {
```

Not a fan of catching generic Exception and this has the potential to mask other errors. Can we throw and catch some particular exceptions? The robot is currently throwing a bunch of IllegalStateException, which might should be changed to something more descriptive, e.g. `OauthLoginException`

---

I could be missing something with the intent of the LabBot, but the error with the story was that the exceptions weren't being captured in TokenExchange bean and were falling through.  So this change captures any error with login to the TokenExchange bean.

---

The concern is that a program error, e.g. NullPointerException, would be classified as "Failed to login". I'd much rather see those types of issues explode the test, rather than be classified as "Bad login"

---

Updated to ripple up program errors.